packages/contracts/contracts/libraries/bridge/CrossDomainEnabled.sol

```diff
@@ -64,22 +61,6 @@ contract CrossDomainEnabled {
     /**********************
      * Internal Functions *
      **********************/
-
-    /**
-     * Gets the messenger, usually from storage. This function is exposed in case a child contract
-     * needs to override.
-     * @return The address of the cross-domain messenger contract which should be used.
-     */
-    function getCrossDomainMessenger()
-        internal
-        virtual
-        returns (
-            ICrossDomainMessenger
-        )
-    {
-        return ICrossDomainMessenger(messenger);
-    }
-
```

Comment:
This feels like a safe change to me, but I'm not sure what the original driver for it was, so it would be good to have @ben-chain take a look.

Reply:
I suspect it is just the more verbose code style that some prefer and I definitely don't 🤷‍♀️

Reply:
well, the comment says:
> This function is exposed in case a child contract needs to override

I'm very doubtful that any one has actually written their own messenger's though.

Reply:
This was probably a premature optimization/generalization where I wanted to allow some wonky inheritance if people wanted it.  This change LGTM.